mayastor/src/bin/mayastor.rs

```diff
@@ -128,7 +128,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             }
         };
 
-    info!("Starting Mayastor version: {}", GIT_VERSION);
+    info!("Starting Mayastor version [CAS-1290 v1]: {}", GIT_VERSION);
```

Comment:
How come we have `CAS-1290` here?

Reply:
ah sorry test message :-) will remove, thank you!